
sim:
  dt: 0.2
  dt_in_sim: 0.2
  image:
    output_image: true
    height: 1500
    width: 400
    m2pix: 100 # pixels per meter
    y_baseline: 100 # pixels up at line y = 0
  world:
    road:
      width: 2.0
  other_duckie_type: "unstable_speed_duckie" # Available: "constant_speed_duckie", "unstable_speed_duckie"


